collections/sice.yaml

```diff
@@ -0,0 +1,79 @@
+  
+  Since the algorithm relies heavily on the sun zenith angle (SZA), the best results for the northern polar regions can be obtained for the periods between May and September (depanding on the location Latitude).  
+
+  To generate the SICE data, navigate to [EDC Browser](https://browser.eurodatacube.com/?zoom=4&lat=74.70645&lng=-45.84855&fromTime=1970-01-01T00%3A00%3A00.000Z&toTime=2021-10-22T14%3A36%3A31.329Z&algorithm=sice), 
+  select the required input parameters and follow the check-out wizard to complete your order. Produced data will be uploaded directly into your JupyterLab environment on EDC.
+Image: sice/thumbnail.png
+Explore: "[Run algorithm in EDC Browser](https://browser.eurodatacube.com/?zoom=4&lat=74.70645&lng=-45.84855&fromTime=1970-01-01T00%3A00%3A00.000Z&toTime=2021-10-22T14%3A36%3A31.329Z&algorithm=sice)" 
+GeographicalCoverage: Polar regions covered by snow
+TemporalAvailability: |  
+ Global:  November 2016 - ongoing
+UpdateFrequency: On-demand
+Attributes:
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description
+        - Name: Type
+          Title: Type      
+      Rows:
+        - Name: aoi
+          Description: Area of interest
+          Type: Polyon or bounding box
+        - Name: Date
+          Description: Date of data retieval
+          Type: String         
+        - Name: Projection
+          Description: Target projection of the produced data. By default the results will be provided in EPSG:4326.
+          Type: String
+Produced results:
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description
+        - Name: Units
+          Title: Units      
+      Rows:
+        - Name: grain_diameter.tif
+          Description: Grain diameter
+          Units: mm
+        - Name: snow_specific_surface_area.tif
+          Description: Snow specific surface area 
+          Units: m2kg-1         
+        - Name: r_TOA_01_valid.tif
+          Description: Top of the Atmosphere Reflectance (OLCI band 1)
+          Units: 
+        - Name: r_TOA_06_valid.tif
+          Description: Top of the Atmosphere Reflectance (OLCI band 6)
+          Units: 
+        - Name: r_TOA_17_valid.tif
+          Description: Top of the Atmosphere Reflectance (OLCI band 7)
+          Units:   
+        - Name: r_TOA_21_valid.tif
+          Description: Top of the Atmosphere Reflectance (OLCI band 21)
+          Units:   
+Pricing: |   
+ Prices are defined based on the selected roi and resolution. Regardles of the size and resolution, the minimal cost for initiating request is 10 Units.
```

Comment:
Small question: Is it clear to the customer what 10 Units mean here? 10 SH processing units/10 EDC units? Maybe we can be a bit more specific on what kind of unit.

Reply:
Changed units to EDC credits as the message says "Estimated algorithm run: 10 credits"

Reply:
Now it's clear, thanks!